index.html

```diff
@@ -0,0 +1,318 @@
+            </p>
+          </a>
+          <a href="">
+            <div style="-webkit-line-clamp: 5">
+              International support for well-known trailblazer continues.
+            </div>
+          </a>
+        </div>
+        <div>
+          <a href=""><img alt="Heatwave or Unprecedented Highs"
+              class="gallery-item-visible gallery-item gallery-item-preloaded" data-hook="gallery-item-image-img"
+              data-idx="1"
+              src="https://static.wixstatic.com/media/c837a6_2c80832d82c9465f9d824be4018b0775~mv2.jpg/v1/fill/w_201,h_151,fp_0.50_0.50,q_90,enc_auto/c837a6_2c80832d82c9465f9d824be4018b0775~mv2.jpg"
+              loading="eager" style="width: 201px; height: 150.75px" /></a>
+          <a href="">
+            <p class="bD0vt9 KNiaIk" style="-webkit-line-clamp: 3">
+              Heatwave or Unprecedented Highs
+            </p>
+          </a>
+          <a href="">
+            <div class="BOlnTh" style="-webkit-line-clamp: 5">
+              Hottest day in South Asia since 1904.
+            </div>
+          </a>
+        </div>
+      </div>
+
+      <div>
+        <img alt="Coastal Cities to Be Inundated by 2050"
+          class="gallery-item-visible gallery-item gallery-item-preloaded" data-hook="gallery-item-image-img"
+          data-idx="0"
+          src="https://static.wixstatic.com/media/c837a6_3a913b38947842bd902b37bcd5548a74~mv2.jpg/v1/fill/w_585,h_585,fp_0.50_0.50,q_90,enc_auto/c837a6_3a913b38947842bd902b37bcd5548a74~mv2.jpg"
+          loading="eager" style="width: 585px; height: 585px" />
+        <p class="bD0vt9 KNiaIk" style="-webkit-line-clamp: 2">
+          Coastal Cities to Be Inundated by 2050
+        </p>
+        <div class="BOlnTh" style="-webkit-line-clamp: 4">
+          New report highlights growing danger to countless cities and
+          populations.
+        </div>
+      </div>
+
+      <div class="main-content-user">
+        <div class="paragraph-name">Opinion</div>
+        <div>
+          <div class="user-info">
+            <img width="32px" height="32px" src="https://img.icons8.com/ios/344/user-male-circle.png" alt="" />
+            <div>
+              <p>Admin</p>
+              <p>Sep 20</p>
+            </div>
+          </div>
+
+          <a href="">
+            <p>Was It Inevitable?</p>
+          </a>
+          <a href="">
+            <div class="BOlnTh" style="-webkit-line-clamp: 4">
+              Non-profit's marine life extinction report is no surprise to
+              environmentalists.
+            </div>
+          </a>
+        </div>
+        <div>
+          <div class="user-info">
+            <img width="32px" height="32px" src="https://img.icons8.com/ios/344/user-male-circle.png" alt="" />
+            <div>
+              <p>Admin</p>
+              <p>Sep 20</p>
+            </div>
+          </div>
+
+          <a href="">
+            <p class="bD0vt9 KNiaIk" style="-webkit-line-clamp: 3; width: calc(100% - 0)">
```

Comment:
remove inline styling use classes instead

spread this notif to other similar structures too